packages/near-fast-auth-signer/src/components/SignMultichain/SignMultichain.tsx

```diff
@@ -172,14 +162,11 @@ function SignMultichain() {
       handleMessage,
     );
 
-    // TODO: test code, delete later
-    // handleMessage({ data: { type: 'multi-chain', data: sampleMessageForBTC } });
+    window.parent.postMessage({ type: 'sign-multichain-load', message: 'SignMultichain page has loaded' }, '*');
```

Comment:
different type convention, we should keep the consistency on use kebab-case or camelCase

Reply:
Good point, changing it :)